11-validation.md

```diff
@@ -14,15 +14,15 @@ This section will help you to validate the workload is exposed correctly and res
 
    ```bash
    # query the Azure Application Gateway Public Ip
-   APPGW_PUBLIC_IP=$(az deployment group show --resource-group rg-enterprise-networking-spokes -n spoke-BU0001A0008 --query properties.outputs.appGwPublicIpAddress.value -o tsv)
-   echo APPGW_PUBLIC_IP: $APPGW_PUBLIC_IP
+   export APPGW_PUBLIC_IP_AKS_BASELINE=$(az deployment group show --resource-group rg-enterprise-networking-spokes -n spoke-BU0001A0008 --query properties.outputs.appGwPublicIpAddress.value -o tsv)
```

Comment:
Why are we exporting now?

Reply:
because https://github.com/mspnp/aks-baseline/blob/main/saveenv.sh save the variables that are exported and end by _AKS_BASELINE. 
Without the export, the variables are not saved. 
That is good when you for some reason need to re-start a deploy process in the middle, you have the variable values till the moment you executed the latest  saveenv.sh.